newrelic-weaver/src/main/java/com/newrelic/weave/MethodCallInlinerAdapter.java

```diff
@@ -7,7 +7,9 @@
 
 package com.newrelic.weave;
 
+import com.newrelic.weave.utils.ReturnInsnProcessor;
 import com.newrelic.weave.utils.WeaveUtils;
+import org.objectweb.asm.tree.*;
```

Comment:
FYI: If you setup Intellij to use the [java-agent-code-style.xml](https://github.com/newrelic/newrelic-java-agent?tab=readme-ov-file#intellij-idea-setup) you can have it clean up all of the important statements for you. We adhere to the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) which   prohibits wildcard imports.